packages/rulesets/src/spectral/functions/utils.ts

```diff
@@ -174,3 +176,16 @@ export function isSchemaEqual(a: any, b: any): boolean {
   }
   return false;
 }
+
+export function createRuleFunctionWithPasses<I = unknown, O = unknown>(fn:(input:I, options:O, ctx: RulesetFunctionContext)=> IFunctionResult[]) {
+  return (input:I, options:O, ctx: RulesetFunctionContext):IFunctionResult[] => {
+    const messsages = fn(input,options,ctx)
+    if (messsages.length === 0) {
+      messsages.push({
+        message: `[Verbose]this is a verbose message to indicate that this rule was passed for specific swagger schema successfully and no fix is needed, please ignore it.`,
```

Comment:
>this is a verbose message to indicate that this rule was passed for specific swagger schema successfully and no fix is needed, please ignore it. [](http://example.com/codeflow?start=27&length=144)

How about this text for the verbose message - "ARM RPC rule passed for path {path}" ?

Reply:
The path is another field of the error object, I think we don't need to duplicate it in the message